PL2/PL2-parser/src/main/java/nl/tudelft/pl2016gr2/parser/controller/AnnotationReader.java

```diff
@@ -0,0 +1,211 @@
+package nl.tudelft.pl2016gr2.parser.controller;
+
+import nl.tudelft.pl2016gr2.model.Annotation;
+import org.apache.poi.openxml4j.exceptions.InvalidFormatException;
+import org.apache.poi.ss.usermodel.Cell;
+import org.apache.poi.ss.usermodel.Row;
+import org.apache.poi.ss.usermodel.Sheet;
+import org.apache.poi.ss.usermodel.Workbook;
+import org.apache.poi.ss.usermodel.WorkbookFactory;
+
+import java.io.File;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Iterator;
+import java.util.List;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+@SuppressWarnings("checkstyle:AbbreviationAsWordInName")
+public class AnnotationReader {
```

Comment:
Initially this was added because a lot of column names in the sheet would cause warnings and I tried to keep the naming the same as much as possible. I've now checked again and only several methods/fields would now need this warning without changing too many names. I've added the warning to the specific cases now instead of the whole class.

I think with the leftover cases it's quite obvious for the developer why the warning is needed now as the names that cause the warning are directly below it.